### PR TITLE
Refactor Blocks/Image/Edit component 

### PR DESCRIPTION
**Pull request by Tishasoumya-02**
#4460

### CODE REVIEW

**Comment by nileshgulia1** [comment]
@Tishasoumya-02 Could you please take care of conflicts?

**Review by nileshgulia1 — APPROVED** [review]

File: src/components/manage/Blocks/Image/Edit.jsx [hunk 1]
```diff
@@ -44,392 +39,340 @@ const messages = defineMessages({
...
+                          <Button
+                            basic
+                            icon
                             onClick={(e) => {
-                              e.target.focus();
-                            }}
-                            onFocus={(e) => {
-                              this.props.onSelectBlock(this.props.id);
+                              e.stopPropagation();
+                              e.preventDefault();
```
Comment (iFlameing) [inline src/components/manage/Blocks/Image/Edit.jsx:271]: We should move this in seperate function.

**Review by sneridagh — CHANGES_REQUESTED** [review]
@Tishasoumya-02 @nileshgulia1 @iFlameing When the size is changed, the image flickers twice or three times. It does not happen in the class based. Could you please take a look? Thanks!

**Comment by Tishasoumya-02** [comment]
This component is already refactored to functional . Closing this PR.

**Pull request closed without merging**